README.md

```diff
@@ -9,7 +9,7 @@ using Django and PostgreSQL for the backend and React & Bootstrap for the fronte
 
 ## Requirements
 
-The application requires Python (version 3.8 or greater), Node.js (Version 16) by using [nvm](https://github.com/nvm-sh/nvm).
+The application requires Python (version 3.8 or greater), Node.js (Version 16 or greater) by using [nvm](https://github.com/nvm-sh/nvm).
```

Comment:
Nvm, maybe omit the usage of nvm as the required version is no longer 16
```suggestion
The application requires Python (version 3.8 or greater), Node.js (Version 16 or greater).
```

Reply:
No nvm needs to be remained since it is the recommended way to install nodejs on ubuntu and mac